packages/esm-appointments-app/src/helpers/excel.ts

```diff
@@ -7,14 +7,22 @@ import { moduleName } from '../constants';
 /**
  * Exports the provided appointments as an Excel spreadsheet.
  * @param {Array<Appointment>} appointments - The list of appointments to export.
+ * @param {Array} rowData - The current rows of the table as rendered in the UI.
  * @param {string} [fileName] - The name of the downloaded file
  */
-export async function exportAppointmentsToSpreadsheet(appointments: Array<Appointment>, fileName = 'Appointments') {
+export async function exportAppointmentsToSpreadsheet(
+  appointments: Array<Appointment>,
+  rowData: Array<any>,
```

Comment:
```suggestion
  rowData: Array<any>,
```
We should have the correct type here